.circleci/rc.yaml

```diff
@@ -11,8 +11,6 @@ plugins:
   - remark-lint-hard-break-spaces
   - remark-lint-blockquote-indentation
   - remark-lint-no-consecutive-blank-lines
-  - - remark-lint-maximum-line-length
```

Comment:
this actually started causing warnings on table lines in spectral 6, but we don't need to change this from the default anyway, so remvoing